src/ssl.c

```diff
@@ -55284,6 +55284,148 @@ int wolfSSL_CTX_set_ecdh_auto(WOLFSSL_CTX* ctx, int onoff)
+ */
+void wolfSSL_CONF_CTX_free(WOLFSSL_CONF_CTX* cctx)
+{
+    WOLFSSL_ENTER("wolfSSL_CONF_CTX_free");
+    
+    if (cctx) {
+        XFREE(cctx, NULL, DYNAMIC_TYPE_OPENSSL);
+    }
+}
+/**
+ * Release WOLFSSL_CONF_CTX instance
+ * @param cctx a pointer to WOLFSSL_CONF_CTX structure to set a pointer
+ *             to WOLFSSL_CTX
+ * @param ctx  a pointer to WOLFSSL_CTX structure to be set
+ * @return none
+ */
+void wolfSSL_CONF_CTX_set_ssl_ctx(WOLFSSL_CONF_CTX* cctx, WOLFSSL_CTX *ctx)
+{
+    WOLFSSL_ENTER("wolfSSL_CONF_CTX_set_ssl_ctx");
+    
+    //sanity check
+    if (cctx == NULL) {
+        WOLFSSL_MSG("cctx is null");
+        return;
+    }
+    
+    if (ctx != NULL) {
+        cctx->ctx = ctx;
+    } else {
+        cctx->ctx = NULL;
+    }
+}
+/**
+ * set flag value into WOLFSSL_CONF_CTX
+ * @param cctx  a pointer to WOLFSSL_CONF_CTX structure to be set
+ * @param flags falg value to be OR'sd
+ * @return OR'd flag value, otherwise 0
+ */
+unsigned int wolfSSL_CONF_CTX_set_flags(WOLFSSL_CONF_CTX* cctx, unsigned int flags)
+{
+    //sanity check
+    if (cctx == NULL) return 0;
+    
+    cctx->flags |= flags;
+    return cctx->flags;
+}
+#ifndef NO_WOLFSSL_STUB
+/**
+ * finish configuration command operation
+ * @param cctx  a pointer to WOLFSSL_CONF_CTX structure to be set
+ * @return WOLFSSL_FAILURE for now
+ */
+int wolfSSL_CONF_CTX_finish(WOLFSSL_CONF_CTX* cctx)
+{
+    WOLFSSL_STUB("wolfSSL_CONF_CTX_finish");
+    (void)cctx;
+    return WOLFSSL_FAILURE;
+}
+/**
+ * send configuration command
+ * @param cctx  a pointer to WOLFSSL_CONF_CTX structure
+ * @param cmd   configuration command
+ * @param value arguments for cmd
+ * @return WOLFSSL_FAILURE for now
+ */
+int wolfSSL_CONF_cmd(WOLFSSL_CONF_CTX* cctx, const char* cmd, const char* value)
+{
+    WOLFSSL_STUB("wolfSSL_CONF_cmd");
+    (void)cctx;
+    (void)cmd;
+    (void)value;
+    return WOLFSSL_FAILURE;
+}
+
+/**
+ * returns a new idex or -1 on failure
+ * @param class index one of CRYPTO_EX_INDEX_xxx
+ * @param argp  parameters to be saved
+ * @param argl  parameters to be saved
+ * @param new_func a pointer to WOLFSSL_CRYPTO_EX_new
+ * @param dup_func a pointer to WOLFSSL_CRYPTO_EX_dup
+ * @param free_func a pointer to WOLFSSL_CRYPTO_EX_free
+ * @return WOLFSSL_FAILURE for now
+ */
+#ifdef HAVE_EX_DATA
+int wolfSSL_CRYPTO_get_ex_new_index(int class_index, long argl, void *argp,
+                                           WOLFSSL_CRYPTO_EX_new* new_func,
+                                           WOLFSSL_CRYPTO_EX_dup* dup_func,
+                                           WOLFSSL_CRYPTO_EX_free* free_func)
+{
+    WOLFSSL_STUB("wolfSSL_CRYPTO_get_ex_new_index");
+    (void)class_index;
+    (void)argl;
+    (void)argp;
+    (void)new_func;
+    (void)dup_func;
+    (void)free_func;
+    return WOLFSSL_FAILURE;
+}
+#endif
+
+/**
+ * retrive p, q and g parameter
+ * @param dh a pointer to WOLFSSL_DH
+ * @param p  a pointer to WOLFSSL_BIGNUM to be obtained dh
```

Comment:
"obtained dh" should be "obtained from dh", applies to next two lines as well.

Reply:
Thanks for pointing it out. Updated